react/features/recording/actions.web.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import React from 'react';
+
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { showNotification } from '../notifications';
+
+import { RecordingLimitNotificationDescription } from './components';
+
+export * from './actions.any';
+
+/**
+ * Signals that a started recording notification should be shown on the
+ * screen for a given period.
+ *
+ * @param {string} streamType - The type of the stream ({@code file} or
+ * {@code stream}).
+ * @returns {showNotification}
+ */
+export function showRecordingLimitNotification(streamType: string) {
+    const isLiveStreaming = streamType === JitsiMeetJS.constants.recording.mode.STREAM;
+
+    return showNotification({
+        description: <RecordingLimitNotificationDescription isLiveStreaming = { isLiveStreaming } />,
+        titleKey: isLiveStreaming ? 'dialog.liveStreaming' : 'dialog.recording'
+    }, 100000);
```

Comment:
Is this value intentionally different than the one for mobile?

Reply:
Not really :) there's one additional zero. Will remove it.

Reply:
Fixed!